cw_solana/lib/solana_client.dart

```diff
@@ -753,19 +761,23 @@ class SolanaWalletClient {
     final blockhash = await _getLatestBlockhash(Commitment.confirmed);
 
     final transaction = SolanaTransaction(
-      payerKey: payerPrivateKey.publicKey().toAddress(),
+      payerKey: payerAddress,
       instructions: [createAssociatedTokenAccount],
       recentBlockhash: blockhash,
+      type: TransactionType.v0,
```

Comment:
what's that used for?

Reply:
It ensures that the transaction being made for is a versioned transaction instead of a legacy transaction